<?xml version="1.0"?>
<!--XML-->
<launch>
    <!-- SETUP ARGUMENTS -->
    <!-- ur10 arguments -->
    <arg name="robot_ip" default="172.16.1.59"/>
    <arg name="robot_name" default="wallace"/>
    <arg name="robot_type" default="ur10"/>

    <!--<include file="$(find ur_robot_driver)/launch/$(arg robot_type)_bringup.launch">
        <arg name="robot_ip" value="$(arg robot_ip)"/>
        <arg name="kinematics_config" value="$(find i3dr_ur10)/calib/$(arg robot_name)_robot_calibration.yaml"/>
    </include>-->

    <include file="$(find ur_modern_driver)/launch/$(arg robot_type)_bringup.launch">
        <arg name="robot_ip" value="$(arg robot_ip)"/>
    </include>

    <include file="$(find ur10_moveit_config)/launch/$(arg robot_type)_moveit_planning_execution.launch">
    </include>

    <include file="$(find ur10_moveit_config)/launch/moveit_rviz.launch">
        <arg name="config" value="true"/>
    </include>

</launch>